pump/storage/vlog.go

```diff
@@ -329,6 +329,8 @@ func (vlog *valueLog) write(reqs []*request) error {
 
 	toDisk := func() error {
 		n, err := curFile.fd.Write(vlog.buf.Bytes())
+		atomic.AddInt64(&vlog.writableLogOffset, int64(n))
```

Comment:
Can this `part of data` be ignored automatically?

Reply:
It will skip such `part of data` if we scan the file